### ISSUE TITLE
Integrate axe-matchers to test suite

### ISSUE THREAD

**Issue by axamei** [pulibrary/orangelight#1225]
[Automated accessibility testing powered by aXe.](https://github.com/dequelabs/axe-matchers)

This project moved away from phantomjs (which is the webdriver we are currently using). Is there a plan to switch over to use the headless chrome webdriver in the near future?

**Comment by tampakis** [comment]
We should have a broader discussion about integrating automated accessibility audits into our CI across applications

**Comment by christinach** [comment]
We added axe-core-rspec https://github.com/pulibrary/orangelight/commit/9b65dbb2c38e7b0f72492ce2a340ef5f6d05961d for accessibility testing
